plugins/source/gcp/policies_v1/queries/dns/zones_without_dnssec.sql

```diff
@@ -0,0 +1,20 @@
+-- select * from gcp_dns_managed_zones
+-- where visibility != 'private'
+-- and ((dnssec_config is null) or (dnssec_config->>'state' = 'off'));
+
+
+INSERT INTO gcp_policy_results (resource_id, execution_time, framework, check_id, title, project_id, status)
+SELECT "id"                                                      AS resource_id,
+       :'execution_time'::timestamp                              AS execution_time,
+       :'framework'                                              AS framework,
+       :'check_id'                                               AS check_id,
+       'Ensure that DNSSEC is enabled for Cloud DNS (Automated)' AS title,
```

Comment:
Agreed with naming - this gets weird as we could do:

zones_without_dnssec_enabled.

From a language perspective, we have:
* With or Without
* Enabled or Not Enabled (Disabled/Off)
* Then Fail or Pass.

Reply:
I think it's probably best to say 
`zones_with_dnssec_disabled`, but technically,  the query is looking for zones where it is not enabled. The difference being that there could be more states than just enabled || disabled